tools/releaseBuild/azureDevOps/templates/linux-authenticode-sign.yml

```diff
@@ -2,6 +2,7 @@ jobs:
 - job: sign_linux_builds
   displayName: Sign all linux builds
   condition: succeeded()
+  timeoutInMinutes: 90
```

Comment:
should this be in a separate PR for backporting?

Reply:
yes.. i will revert this change.

Reply:
Reverted